prompting/baseminer/miner.py

```diff
@@ -108,7 +108,11 @@ def __init__(self, config=None, axon=None, wallet=None, subtensor=None):
             bt.logging.info(f"Running miner on uid: {self.my_subnet_uid}")
 
         # The axon handles request processing, allowing validators to send this process requests.
-        self.axon = axon or bt.axon(wallet=self.wallet, port=self.config.axon.port)
+        self.axon = axon or bt.axon(
+            wallet=self.wallet,
+            port=self.config.axon.port,
+            external_ip=self.config.axon.external_ip,
```

Comment:
Where is the `axon.external_ip` coming from exactly? I did a quick lookup for `axon.external_ip` in vscode and it returned no results. Not sure if we have this parameter already 🤔

Reply:
It should be added by the default bittensor argparser for axon.

E.g.: https://github.com/opentensor/bittensor/blob/master/bittensor/axon.py#L448

If [this](https://github.com/opentensor/text-prompting/blob/main/prompting/baseminer/config.py#L238) line is called in instantiating the miner base class, then the parser already has `axon.port` and there is no need for it to be added a 2nd time in `text-prompting`.

Is this what you're referring to @camfairchild ?